src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewService.ts

```diff
@@ -62,6 +62,24 @@ export interface IPositronNotebookOutputWebviewService {
 		viewType?: string,
 	): Promise<INotebookOutputWebview | undefined>;
 
+	/**
+	 * Create a new notebook output webview from a series of output messages.
+	 *
+	 * This is useful for situations where a plot may have dependencies that are provided by
+	 * separate messages.
+	 *
+	 * @param runtime The runtime that emitted the output
+	 * @param outputs The messages to be sent into webview. The final message that triggered the
+	 * plotting should be the final element of the array.
```

Comment:
Nit: Since the last message is expected to have special treatment, maybe its better to have a separate argument for that one?

Reply:
Yeah, that's what I did in the internal methods of the holoviews service but I attempted to minimize changes here. That being said that attempt does make it more confusing. i'll update it!